SimTKcommon/adolcTarget/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+## This whole directory exists just so I could define this extra
+## preprocessor value.
+
+add_definitions(-DSimTK_SimTKCOMMON_BUILDING_SHARED_LIBRARY)
+
+if(BUILD_UNVERSIONED_LIBRARIES)
+    add_library(${ADOLC_TARGET} SHARED
+            ${SOURCE_FILES} ${SOURCE_INCLUDE_FILES}
+            ${API_ABS_INCLUDE_FILES})
+
+    add_dependencies(${ADOLC_TARGET} PlatformFiles)
+
+    # TODO
+    target_include_directories(${ADOLC_TARGET} PRIVATE "${ADOLC_INCLUDES}")
```

Comment:
Thanks for pointing out this TODO. I think we should remove the `target_include_directories(...)` lines for now (there's one below as well). The `include_directories("${ADOLC_INCLUDES}")` in the root CMakeLists.

This line is the preferred way to set include directories, but requires CMake 3.0. Our minimum required CMake version is 2.8 right now. So we'll just make this change later on.